LogMonitor/src/LogMonitor/sample-config-files/IIS/LogMonitorConfig.json

```diff
@@ -22,6 +22,12 @@
         "filter": "*.log",
         "includeSubdirectories": true
       },
+      {
+        "type": "File",
+        "directory": "c:\\wwwroot",

```

Comment:
The default location for IIS Log files is %SystemDrive%\inetpub\logs\LogFiles. Any reason why you had this as wwwroot?